client/components/gsuite/gsuite-new-user-list/index.tsx

```diff
@@ -22,9 +22,10 @@ interface Props {
 	onUsersChange: ( users: NewUser[] ) => void;
 	onReturnKeyPress: ( event: Event ) => void;
 	users: NewUser[];
+	validatedMailboxUuids?: string[];
 }
 
-const GSuiteNewUserList: FunctionComponent< Props > = ( {
+const GSuiteNewUserList: FunctionComponent< GSuiteNewUserListProps > = ( {
```

Comment:
Nit: it may be simpler to define this without using `FunctionComponent`. But that's not blocking.
```suggestion
const GSuiteNewUserList = ( {
  [props...]
}: GSuiteNewUserListProps ): ReactElement => {
```